bootstrap/eks/api/v1alpha4/eksconfigtemplate_types.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha4
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EKSConfigTemplateSpec defines the desired state of EKSConfigTemplate
+type EKSConfigTemplateSpec struct {
+	Template EKSConfigTemplateResource `json:"template"`
+}
+
+// EKSConfigTemplateResource defines the Template structure
+type EKSConfigTemplateResource struct {
+	Spec EKSConfigSpec `json:"spec,omitempty"`
+}
+
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:path=eksconfigtemplates,scope=Namespaced,categories=cluster-api
+// +kubebuilder:storageversion
```

Comment:
And maybe this?

Reply:
We need this so that we store the api in this version.
Removed this from v1alpha3 types.